### PR TITLE
Update messages_de_DE.properties

### PR DESCRIPTION
**Pull request by pixma140**
Updated translations for German language.

# Description

I updated the translations for German language.

Closes no issue.

## Checklist

- [x] I have read the [Contribution Guidelines](https://github.com/Stirling-Tools/Stirling-PDF/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas - I guess this does not apply for translations
- [x] My changes generate no new warnings
- [ ] I have read the section [Add New Translation Tags](https://github.com/Stirling-Tools/Stirling-PDF/blob/main/HowToAddNewLanguage.md#add-new-translation-tags) (for new translation tags only) - not checked since it is no new translation tag

### CODE REVIEW

**Comment by Ludy87** [comment]
Did you run the translation through Google Translate? There are strings that shouldn't be translated one-to-one in German.

File: src/main/resources/messages_de_DE.properties [hunk 1]
```diff
@@ -114,21 +114,21 @@ pipelineOptions.validateButton=Validieren
 ########################
 #  ENTERPRISE EDITION  #
 ########################
-enterpriseEdition.button=Upgrade to Pro
-enterpriseEdition.warning=This feature is only available to Pro users.
-enterpriseEdition.yamlAdvert=Stirling PDF Pro supports YAML configuration files and other SSO features.
-enterpriseEdition.ssoAdvert=Looking for more user management features? Check out Stirling PDF Pro
+enterpriseEdition.button=Rüsten Sie auf Pro auf
```
Comment (Ludy87) [inline src/main/resources/messages_de_DE.properties]: Auf Pro-Version umsteigen

File: src/main/resources/messages_de_DE.properties [hunk 2]
```diff
@@ -114,21 +114,21 @@ pipelineOptions.validateButton=Validieren
...
+enterpriseEdition.warning=Diese Funktionalität ist nur verfügbar für Benutzer mit der Pro Version.
```
Comment (Ludy87) [inline src/main/resources/messages_de_DE.properties]: Diese Funktion ist nur für Pro-Nutzer verfügbar.

**Comment by Ludy87** [comment]
There are still a few improvements, but I'll leave those up to you ;-)

**Comment by pixma140** [comment]
@Ludy87 thank you for your comments. 

> Did you run the translation through Google Translate? There are strings that shouldn't be translated one-to-one in German.

No, I just did the translations by myself. Your additions sound good to me, whereas mine partly sound very "steinern".

> There are still a few improvements, but I'll leave those up to you ;-)

I did some more translations and edits now. I hope they suit you better. I did everything here in good faith and based on the best knowledge available to me.

**Review by Ludy87 — APPROVED** [review]

**Review by Frooodle — APPROVED** [review]

**Comment by pixma140** [comment]
Hello @Frooodle and/or @Ludy87,
this was one of my very first ever contributions to any repository. I never worked with translations before and don´t know how in GitHub the actions work to build the language completeness indicator seen here: https://hub.docker.com/r/stirlingtools/stirling-pdf

Can you tell me, is there an easy way to find out which strings in the translation are missing to achieve the 100% translation?

Help appreciated! :)

**Pull request merged**